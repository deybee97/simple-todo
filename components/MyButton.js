import React from "react"
import { Button } from "react-native"

const MyButton = ()=> <Button title="Button" onPress={()=>{}}/>

 export default MyButton