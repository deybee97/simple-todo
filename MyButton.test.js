import MyButton from "./components/MyButton.js"
import renderer from "react-test-renderer"


describe('MyButton', () => {
  it('renders',()=>{

    const button = renderer.create(<MyButton/>).toJSON()
    expect(button).toMatchSnapshot()

  })
})
