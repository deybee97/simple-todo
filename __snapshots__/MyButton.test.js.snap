// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyButton renders 1`] = `null`;
